frontend/src/components/ThemedButtons/ThemedButton.css

```diff
@@ -5,11 +5,16 @@
 	position: relative;
 	border: none;
 	border-radius: 0.625rem;
-	padding: 0.5rem;
+	padding: 0.45rem;
 	background-color: var(--action-button-background-colour);
 	color: var(--action-button-text-colour);
 	font-size: 1rem;
 	font-weight: 600;
+	border: 1px solid var(--action-button-background-colour);
```

Comment:
You have border: none earlier, so remove that please.

Also, 1px will not scale, so prefer rems.